src/libs/test.ts

```diff
@@ -0,0 +1,37 @@
+counter = 0;
+
+function incrementCounter() {
+  counter++;
+}
+
+function sum(a: number, b: number) {
+  return a + b;
+}
+
+function multiply(a: number, b: number) {
+  return a * b;
+}
+
+const apiKey = process.env.API_KEY;
```

Comment:
@trag-bot Potential security risk if the API key is not properly secured or if it is logged or exposed inappropriately.